tests/e2e/util.go

```diff
@@ -5746,3 +5746,75 @@ func waitAndGetContainerID(sshClientConfig *ssh.ClientConfig, k8sMasterIP string
 	}
 	return containerId, waitErr
 }
+
+// assignPolicyToWcpNamespace assigns a set of storage policies to a wcp namespace
+func assignPolicyToWcpNamespace(client clientset.Interface, ctx context.Context,
+	namespace string, policyNames []string) {
+	vcIp := e2eVSphere.Config.Global.VCenterHostname
+	vcAddress := vcIp + ":" + sshdPort
+	sessionId := createSessionIdForVc()
+
+	curlStr := ""
+	policyNamesArrLength := len(policyNames)
+	limit, err := strconv.Atoi(defaultrqLimit)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	limit *= 953 //to convert gb to mebibytes
+	if policyNamesArrLength >= 1 {
+		curlStr += fmt.Sprintf(`{ "limit": %d, "policy": "%s"}`, limit, e2eVSphere.GetSpbmPolicyID(policyNames[0]))
+	}
+	if policyNamesArrLength >= 2 {
+		for i := 1; i < policyNamesArrLength; i++ {
+			profileID := e2eVSphere.GetSpbmPolicyID(policyNames[i])
+			curlStr += "," + fmt.Sprintf(`{ "limit": %d, "policy": "%s"}`, limit, profileID)
+		}
+	}
+
+	curlCmd := fmt.Sprintf(`curl -k -X PATCH 'https://%s/api/vcenter/namespaces/instances/%s' -H `+
+		`'vmware-api-session-id: %s' -H 'Content-type: application/json' -d `+
+		`'{ "access_list": [ { "domain": "", "role": "OWNER", "subject": "", "subject_type": "USER" } ], `+
+		`"description": "", "resource_spec": { }, "storage_specs": [ %s ], `+
+		`"vm_service_spec": { } }'`, vcIp, namespace, sessionId, curlStr)
+
+	framework.Logf("Running command: %s", curlCmd)
```

Comment:
that is not enough, curl will be successful even if the restapi fails in many cases, use -x option in curl to see the return status and verify it

Reply:
Done